### PR TITLE
Block Q/A & Sidebar refactor

### PR DESCRIPTION
**Pull request by dzucconi**
Closes "Block view" items on https://github.com/aredotna/ervell/issues/580

And generally refactors the sidebar styling and DOM structure.

### CODE REVIEW

File: apps/block/templates/index.jade [hunk 1]
```diff
@@ -1,14 +1,13 @@
-extends ../../../components/layout/index
+extends ../../../components/layout/index--infinite
```
Comment (dzucconi) [inline apps/block/templates/index.jade]: Should probably rename this but 'footerless' is also weird.

File: apps/block/templates/sidebar.jade [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.BlockSidebar.js-block-sidebar( class=(hideSidebar === 'true' ? 'BlockSidebar--hidden' : '') )
+  .BlockSidebar__content
+    section.BlockSidebar__section
+      //- Conditional here prevents content shifting when logged out
```
Comment (dzucconi) [inline apps/block/templates/sidebar.jade]: YES.

File: apps/block/templates/sidebar.jade [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        if block.createdAtAgo() !== block.updatedAtAgo()
+          time( datetime= block.get('updated_at'), title= block.get('updated_at') )
+            | Last updated #{block.updatedAtAgo()}
+
+
+    section.BlockSidebar__section
+      header.BlockSidebar__header
+        | Info
+
+      //- Conditional here prevents content shifting when logged out
```
Comment (dzucconi) [inline apps/block/templates/sidebar.jade]: YES.

File: apps/block/templates/sidebar.jade [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      if (user && user.id === block.get('user').id) || block.get('description')
+        .BlockSidebar__description( id="attribute-description_#{block.id}" )
+          //- Non-breaking space here prevents content shifting when logged in
```
Comment (dzucconi) [inline apps/block/templates/sidebar.jade]: YES.

File: apps/block/templates/sidebar.jade [hunk 5]
```diff
@@ -0,0 +1,51 @@
+.BlockSidebar.js-block-sidebar( class=(hideSidebar === 'true' ? 'BlockSidebar--hidden' : '') )
+  .BlockSidebar__content
+    section.BlockSidebar__section
+      //- Conditional here prevents content shifting when logged out
+      if (user && user.id === block.get('user').id) || block.get('title')
+        h5.BlockSidebar__title( id="attribute-title_#{block.id}" )
+          | #{_.unescape(block.get('title'))}&nbsp;
+
+      .BlockSidebar__timestamps
+        time( datetime= block.get('created_at'), title= block.get('created_at')  )
```
Comment (dzucconi) [inline apps/block/templates/sidebar.jade]: The underappreciated `time` tag.

File: apps/block/templates/index.jade (hunk 1 above)
Comment (broskoski) [inline apps/block/templates/index.jade]: Agreed, I don't mind infinite

File: apps/block/templates/sidebar.jade (hunk 2 above)
Comment (broskoski) [inline apps/block/templates/sidebar.jade]: oo

File: apps/block/templates/sidebar.jade (hunk 3 above)
Comment (broskoski) [inline apps/block/templates/sidebar.jade]: Ooo

File: components/block_collection/stylesheets/block_collection.styl [hunk 6]
```diff
@@ -1,16 +1,5 @@
-/* GENERAL */
-borderColor(color)
-  rgba(color, 30%)
```
Comment (dzucconi) [inline components/block_collection/stylesheets/block_collection.styl]: I removed this guy for a few reasons: it's weird that we have to use snakecase to work around a naming conflict, it's one line so it's hardly worth wrapping, and I think the thing we're really abstracting is just the 30%.

File: apps/block/templates/sidebar.jade (hunk 4 above)
Comment (broskoski) [inline apps/block/templates/sidebar.jade]: Oooo

File: apps/block/stylesheets/sidebar.styl [hunk 7]
```diff
@@ -1,96 +1,103 @@
...
-      p
-        p 14px
-        margin-top 0
-        word-break break-word
-      li
-        word-break break-word
-      a
-        font-weight bold
+
+  &__provenance
```
Comment (broskoski) [inline apps/block/stylesheets/sidebar.styl]: So fancy

File: apps/block/stylesheets/sidebar.styl [hunk 8]
```diff
@@ -1,96 +1,103 @@
...
-    color lightgray
-    font-weight bold
-    font-size sb_font_size
-    cursor pointer
-    text-align right
-    display block
+    collapsed-margins(4em)
 
-  &__info
-    &__title
```
Comment (dzucconi) [inline apps/block/stylesheets/sidebar.styl]: http://getbem.com/faq/#css-nested-elements

File: apps/block/stylesheets/sidebar.styl (hunk 7 above)
Comment (dzucconi) [inline apps/block/stylesheets/sidebar.styl]: I knew you'd comment on that.

File: apps/block/templates/content_inner.jade [hunk 9]
```diff
@@ -0,0 +1,30 @@
+if block.has('image') && block.get('class') !== 'Media'
+  .block-img
+    a( href= block.getSourceUrl(), target='_blank' )
+      img( src= block.resizeImage(1000, 1000, 'large') )
```
Comment (dzucconi) [inline apps/block/templates/content_inner.jade]: I'm just realizing now that this `resizeImage` method doesn't do anything? I'm assuming we used to use the resizing proxy here?

File: apps/block/stylesheets/sidebar.styl (hunk 8 above)
Comment (broskoski) [inline apps/block/stylesheets/sidebar.styl]: Good to know

**Comment by broskoski** [comment]
LGTM, merge when ready

**Pull request merged**